src/gui/QvisExpressionsWindow.C

```diff
@@ -1086,6 +1087,9 @@ QvisExpressionsWindow::apply()
 //    Brad Whitlock, Thu Oct 27 14:42:53 PDT 2011
 //    Make the name active so we can change it.
 //
+//    Eddie Rusu, Wed Jun 24 15:46:57 PDT 2020
+//    Adding and deleting expressions will not notify the viewer and will still
+//    update the expressions window.
```

Comment:
I'm not sure I understand your modification comment. I think it describes the bug you are fixing, not the solution.

Reply:
Okay, how about now?

Reply:
Your new comment is much clearer.